src/test/java/com/o19s/es/ltr/query/StoredLtrQueryIT.java

```diff
@@ -55,14 +58,24 @@ public void testFullUsecase() throws Exception {
         addElement(new StoredFeature("feature2", Collections.singletonList("query"), "mustache",
                 QueryBuilders.matchQuery("field2", "{{query}}").toString()));
 
+        addElement(new StoredDerivedFeature("feature3", "feature1 * 2"));
```

Comment:
since you change the test you need to add new assertions or change some existing ones.
With or without the derived feature we expect the same thing, e.g. your code can silently fail and we won't notice the failure.